test/atomic_enum_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class TestAtomicEnum < MiniTest::Unit::TestCase
```

Comment:
This class should be called `AtomicEnumTest` for consistent naming (the file is called `atomic_enum_test.rb`)